docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

```diff
@@ -1239,16 +1242,49 @@ Only one of the following parameter is allowed, also when one of them is configu
 [source,xml]
 ----
 "test_parameters": {
-   "qr-code": "MT:-24J042C00KA0648G00"
-   "manual-code": "34970112332"
+  "qr-code": "MT:-24J042C00KA0648G00"
+  "manual-code": "34970112332"
 }
 ----
 +
 WARNING: This is an invalid configuration. TH will not accept both parameters set at same time.
 
 On completion of the "network" and the "dut_config" configuration, select the *Update* and then *Create* button to create the Test Project. 
 
-===== Upload PICS File
+==== PIXIT Support
+
+PIXIT type parameters must be filled in the *test_parameters* section.
```

Comment:
Since this section is from the document, please replace with a `ASCIIDOC link` to that section instead.